src/js/components/NodesGridDials.js

```diff
@@ -82,12 +82,12 @@ var NodesGridDials = React.createClass({
     var props = this.props;
 
     if (props.showServices) {
-      config = this.getServiceSlicesConfig(node);
+      config = this.getServiceSlicesConfig(node) || [];
```

Comment:
looking at `getServicesSlicesConfig`, i think it will always return at least an empty array, so this is unecessary